pruntime/enclave/src/lib.rs

```diff
@@ -1425,8 +1426,30 @@ fn parse_events(block_with_events: &BlockHeaderWithEvents, ecdh_privkey: &EcdhKe
                         };
                         handle_execution(state, &pos, who.clone(), *contract_id, payload, *num, ecdh_privkey);
                     },
-                    _ => {
+                    phala::RawEvent::TransferToTee(_, _) => {
+                        state.contract2.handle_event(evt.event.clone());
+                    },
+                    phala::RawEvent::TransferToChain(_, _, _) => {
                         state.contract2.handle_event(evt.event.clone());
+                    },
+                    phala::RawEvent::TransferTokenToTee(_, _, _) => {
+                        state.contract3.handle_event(evt.event.clone());
+                    },
+                    phala::RawEvent::TransferTokenToChain(_, _, _, _) => {
+                        state.contract3.handle_event(evt.event.clone());
+                    },
+                    phala::RawEvent::TransferXTokenToChain(_, _, _, _) => {
+                        state.contract3.handle_event(evt.event.clone());
+                    },
```

Comment:
```suggestion
                    phala::RawEvent::TransferTokenToTee(_, _, _) | phala::RawEvent::TransferTokenToChain(_, _, _, _) | phala::RawEvent::TransferXTokenToChain(_, _, _, _) => {
                        state.contract3.handle_event(evt.event.clone());
                    },
```

Reply:
Also take care of the indent: https://doc.rust-lang.org/1.0.0/style/style/whitespace.html